spec/rspec/support/differ_spec.rb

```diff
@@ -468,6 +468,17 @@ def inspect; "<BrokenObject>"; end
             expect(diff).to be_diffed_as(expected_diff)
           end
         end
+
+        context 'when expected or actual is false' do
+          it 'generates a diff' do
+            expected = true
+            actual = false
+            expected_diff = "\n@@ -1,2 +1,2 @@\n-false\n+true\n"
+
+            diff = differ.diff(expected,actual)
+            expect(diff).to be_diffed_as(expected_diff)
```

Comment:
This doesn't actually cover both scenarios, rather than setting actual / expected false, do you just want to cover that diffing `true, false` and `false, true` both produce diffs?